amida-tech/indaba-client
# Feature/inba 883 jf remove grommet button

#### What does this PR do?
Removes all references to Grommet buttons, standardizing most buttons with the new `_buttons.scss`. Also removed a number of anonymous functions and fixed a language loading issue.

#### Related JIRA tickets:
INBA-883. (https://jira.amida-tech.com/browse/INBA-883)

#### How should this be manually tested?
Do a regression test against these components and verify that buttons continue to function as intended. 

1. All buttons in the Create Project Wizard.
2. Project Management, the Subjects tab.
3. Project Management, the Users tab.
4. The "Saving Setting" button on the Profile page.
5. The Login page's button.
6. The "Create New" button and "Logout" link for the navigation bar. **Note: The create project wizard has temporary !important tags on a few stylings. I've created a ticket to remove it following the removal of Grommet's modal.**
7. The buttons in the Messages inbox, an when creating a new message.

#### Background/Context
#### Screenshots (if appropriate):
